taxi/tests/test_admin.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+
+class AdminSiteTest(TestCase):
+    def setUp(self) -> None:
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin.user",
+            password="admin1234"
+        )
+        self.client.force_login(self.admin_user)
+        self.driver = get_user_model().objects.create_user(
+            username="test_driver",
+            password="test12345",
+            license_number="TEST01234"

```

Comment:
but it is incorrect `license_number`... Should be only 3 letters... Did you run your tests? Do they work? 🥲

Reply:
![Screenshot 2022-11-21 134300](https://user-images.githubusercontent.com/105078166/203043240-67ecfbdc-e7de-402f-8f4e-b34786040d1b.png)
I guess its not how it should work 🥲

Reply:
But I guess if it is required to validate license_number in admin panel `clean ` and `save ` must be implemented in the models but it will brake all other tests.

Reply:
Just to be sure

![Screenshot 2022-11-21 142002](https://user-images.githubusercontent.com/105078166/203054084-816f00bc-0064-412d-8cb6-d0c06f3dbc5c.png)

Reply:
hmm, but that is really strange